start_amqp.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Run both boulder and cfssl, using test configs.
+if type realpath >/dev/null 2>/dev/null; then
+  cd $(realpath $(dirname $0))
+fi
+
+run_boulder() {
+  local prog=$1
+  if [ "x$BOULDER_CONFIG" == "x" ] ; then
```

Comment:
Not necessary for this change, but there's a cute bash-ism you may find useful:

```
BOULDER_CONFIG=${BOULDER_CONFIG:-test/boulder-config.json}
```

Makes default vars that much easier.